seeds/web-demo/package.json

```diff
@@ -8,12 +8,13 @@
   "types": "dist/src/index.d.ts",
   "type": "module",
   "scripts": {
-    "test": "FORCE_COLOR=1 ava",
+    "test": "echo test",
```

Comment:
I'm a bit confused about the `echo test` used here. Am I missing something?

Reply:
I am not sure what is the test framework that will be used for this project in the future. Since ava throwed some issues due to absence of test cases, I have replaced it with simple a echo command for now for some time. I will update the command once I get an idea on how to build and execute test cases on the same.

Reply:
This seems fine for now.